modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/autocomplete/autocomplete.js

```diff
@@ -30,6 +30,8 @@
 
 	var AutoCompleteCKEditor = function () {};
 
+	var BR_TAG = 'BR';
```

Comment:
This is just used once, so taking the var out might hurt readability more than help anything else. We no longer have the hard rule of "defining every string in a variable", so I'd say we can simply inline this in place.